### PR TITLE
*: add specific Helm version

### PR DESCRIPTION
**Pull request by oraluben**
### What is changed, added or deleted? 

Specific Helm version.

### What is the related PR or file link(s)? 

- [x] Reference link(s): [#1818](https://github.com/pingcap/docs/issues/1818)
- [ ] This PR is to align with:
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [x] All active versions: dev, v3.0, v2.1, v3.1
- [ ] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [ ] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

File: dev/how-to/get-started/deploy-tidb-from-docker-compose.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ To customize the cluster, you can edit the `docker-compose.yml` file directly. I
 
 1. Install Helm.
 
-    [Helm](https://helm.sh) can be used as a template rendering engine. To use Helm, you only need to download its binary file:
+    [Helm](https://helm.sh) can be used as a template rendering engine. Currently we only support Helm version >= 2.9.0 and < 3.0.0.
```
Comment (lilin90) [inline dev/how-to/get-started/deploy-tidb-from-docker-compose.md]:
```suggestion
    [Helm](https://helm.sh) can be used as a template rendering engine. Make sure your Helm version >= 2.9.0 and < 3.0.0. To use Helm, you only need to download its binary file:
```

File: dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ tidb-ctl schema in mysql
 
 ## Use Helm
 
-[Helm](https://helm.sh/) is a package management tool for Kubernetes. Refer to [Helm Documentation](https://github.com/helm/helm#install) to install Helm. The steps are as follows:
+[Helm](https://helm.sh/) is a package management tool for Kubernetes. Currently we only support Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md]:
```suggestion
[Helm](https://helm.sh/) is a package management tool for Kubernetes. Make sure your Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```

File: dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md [hunk 3]
```diff
@@ -148,7 +148,12 @@ tidb-ctl schema in mysql
     curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
     ```
 
-    Or if on macOS, you can use homebrew to install Helm via the `brew install kubernetes-helm` command.
+    Or if on macOS, you can use homebrew to install Helm via the command below:
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md]:
```suggestion
    Or if on macOS, you can use Homebrew to install Helm via the following command:
```

File: v2.1-legacy/op-guide/docker-compose.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ To customize the cluster, you can edit the `docker-compose.yml` file directly. I
 
 1. Install Helm.
 
-    [Helm](https://helm.sh) can be used as a template rendering engine. To use Helm, you only need to download its binary file:
+    [Helm](https://helm.sh) can be used as a template rendering engine. Currently we only support Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```
Comment (lilin90) [inline v2.1-legacy/op-guide/docker-compose.md]:
```suggestion
    [Helm](https://helm.sh) can be used as a template rendering engine. Make sure your Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```

File: v3.0/tidb-in-kubernetes/reference/tools/in-kubernetes.md [hunk 5]
```diff
@@ -139,7 +139,7 @@ tidb-ctl schema in mysql
 
 ## Use Helm
 
-[Helm](https://helm.sh/) is a package management tool for Kubernetes. Refer to [Helm Documentation](https://github.com/helm/helm#install) to install Helm. The steps are as follows:
+[Helm](https://helm.sh/) is a package management tool for Kubernetes. Currently we only support Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/reference/tools/in-kubernetes.md]:
```suggestion
[Helm](https://helm.sh/) is a package management tool for Kubernetes. Make sure your Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```

File: v3.1/tidb-in-kubernetes/reference/tools/in-kubernetes.md [hunk 6]
```diff
@@ -138,7 +138,7 @@ tidb-ctl schema in mysql
 
 ## Use Helm
 
-[Helm](https://helm.sh/) is a package management tool for Kubernetes. Refer to [Helm Documentation](https://github.com/helm/helm#install) to install Helm. The steps are as follows:
+[Helm](https://helm.sh/) is a package management tool for Kubernetes. Currently we only support Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```
Comment (lilin90) [inline v3.1/tidb-in-kubernetes/reference/tools/in-kubernetes.md]:
```suggestion
[Helm](https://helm.sh/) is a package management tool for Kubernetes. Make sure your Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```

File: dev/how-to/get-started/deploy-tidb-from-docker-compose.md [hunk 7]
```diff
@@ -67,7 +67,8 @@ To customize the cluster, you can edit the `docker-compose.yml` file directly. I
 
 1. Install Helm.
 
-    [Helm](https://helm.sh) can be used as a template rendering engine. To use Helm, you only need to download its binary file:
+    [Helm](https://helm.sh) can be used as a template rendering engine. Make sure your Helm version >= 2.9.0 and < 3.0.0. To use Helm, you only need to download its binary file:
+    To use Helm, you only need to download its binary file:
```
Comment (ran-huang) [inline dev/how-to/get-started/deploy-tidb-from-docker-compose.md]: This sentence repeats with the previous one.

**Pull request merged**